src/tools/website-components/Playground.jsx

```diff
@@ -1,100 +1,200 @@
 /* eslint-disable import/no-extraneous-dependencies */
-import React, { Fragment } from 'react';
-import classnames from 'classnames';
+import React, { Fragment, PureComponent } from 'react';
 import ReactDOMServer from 'react-dom/server';
 import PropTypes from 'prop-types';
 import Prism from 'prismjs';
 import { html as beautifyHtml } from 'js-beautify';
+import { queryOne } from '@ecl/ec-base/helpers/dom';
 
+import iconSprite from '@ecl/ec-resources-icons/dist/sprites/icons.svg';
 import Iframe from './Showcase/Iframe';
 import styles from './Playground.scss';
 
-const Playground = ({
-  frameHeight,
-  playgroundLink,
-  system,
-  selectedKind,
-  selectedStory,
-  showFrame,
-  hideDemo,
-  children,
-}) => {
-  if (!children) return null;
-
-  const playgroundUrl =
-    playgroundLink ||
-    encodeURI(
-      `/storybook/${system}/index.html?selectedKind=${selectedKind}&selectedStory=${selectedStory}&stories=1`
+class Playground extends PureComponent {
+  constructor(props) {
+    super(props);
+    this.playgroundRef = React.createRef();
+
+    // Parameters
+    this.showcaseCodeSelector = '[data-ecl-showcase-code]';
```

Comment:
why not use a reference directly instead of using a `data-*` attribute selector?